README.md

```diff
@@ -20,7 +20,7 @@
 
 ## Overview
 
-The sqlserver module installs and manages Microsoft SQL Server 2012, 2014 and 2016 on Windows systems.
+The sqlserver module installs and manages Microsoft SQL Server 2012, 2014, 2016 and 2017 on Windows systems.
```

Comment:
On the one hand, we'll have to keep updating this list forever. On the other hand, I'm inclined to believe that being super specific about supported versions is probably a good thing.

Reply:
It started super specific, so we're carrying on. You can create a DOCS ticket to change verbage if you want